contrib/runners/orquesta_runner/orquesta_runner/runner.yaml

```diff
@@ -4,6 +4,11 @@
   enabled: true
   name: orquesta
   runner_module: orquesta_runner
+  runner_parameters:
+    notify:
```

Comment:
For action chain and mistral we also support ``skip_notify`` runner parameter, right?

It seems like a good idea to also support that here for consistency reasons, or?

Reply:
Nope, it's not a good idea. The skip notify and notify conflicts. `skip_notify` assumes notification is default on all task completion.  `notify` is the opposite.  @LindsayHill agree with me here that we should not opt-in folks into task notification.  it's a real hassle to have to explicitly turn off notification for tasks.  it's more common to don't want notification.

Reply:
I agree about that behavior as well - I was never a fan of opt-out, should have been opt-in.

Having said that, we just need to be explicit about that "behavior change" in the docs / upgrade notes (well, it's not really a behavior change, but inconsistency between 3 runners).

Reply:
Let's please document that behavior somewhere, besides that, LGTM :+1:

Reply:
Yep. I will document this.